src/loaders/IVolumeLoader.ts

```diff
@@ -34,6 +35,8 @@ export class VolumeDims {
  */
 export type PerChannelCallback = (volume: Volume, channelIndex: number) => void;
 
+export type RawChannelDataCallback = (channelIndex: number, data: Uint8Array, atlasDims?: [number, number]) => void;
```

Comment:
is atlasDims the number of rows and columns to partition the Uint8Array into?

Reply:
`atlasDims` is the width/height of the atlas in pixels. It's basically there to account for the fact that `OMEZarrLoader` gets data into a `Volume` via `setChannelDataFromVolume`, but `JsonImageInfoLoader` gets data in with `setChannelDataFromAtlas`. The latter has arguments for `atlasWidth` and `atlasHeight`, so if `atlasDims` is present in the callback then `ThreadableVolumeLoader` calls it with those arguments, and if it isn't it calls `setChannelDataFromVolume`.